timur/test/javascript/utils/query_graph.spec.ts

```diff
@@ -57,4 +58,19 @@ describe('QueryGraph', () => {
       ['labor', 'prize']
     ]);
   });
+
+  describe('for xcrs1 models', () => {
+    const models = require('../fixtures/xcrs1_magma_metadata.json').models;
```

Comment:
This behavior always worked, but I wrote the test as a first step to diagnosing the issue.  I left the test because, well, it's still a useful regression test and it runs fast, so.